Jenkinsfile

```diff
@@ -81,7 +81,7 @@ pipeline {
                         sh 'echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin || true'
                         sh 'make setup-kong-build-tools'
                         sh 'make RESTY_IMAGE_BASE=debian KONG_TEST_CONTAINER_TAG="${GIT_BRANCH##*/}-debian" ADDITIONAL_TAG_LIST="${GIT_BRANCH##*/} ${GIT_COMMIT}" RESTY_IMAGE_TAG=11 release-docker-images'
-                        sh 'make RESTY_IMAGE_BASE=ubuntu KONG_TEST_CONTAINER_TAG="${GIT_BRANCH##*/}-ubuntu" RESTY_IMAGE_TAG=20.04 release-docker-images'
+                        sh 'make RESTY_IMAGE_BASE=ubuntu KONG_TEST_CONTAINER_TAG="${GIT_BRANCH##*/}-ubuntu" RESTY_IMAGE_TAG=22.04 release-docker-images'
```

Comment:
Is this removal intended? Could we leave both to avoid the breaking change? The 20.04 package is used, e.g., in the perf testing framework. @fffonion thoughts?

Reply:
It is intended. Currently we only release per-commit docker images of the most recent LTS so Debian 11 / Alpine 3 therefore Ubuntu 22.04